networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -82,3 +84,13 @@ def test_to_vertex_cover(self):
         matching = maximum_matching(self.graph)
         vertex_cover = to_vertex_cover(self.graph, matching)
         self.check_vertex_cover(vertex_cover)
+
+
+def test_eppstein_matching():
+    # Special test
```

Comment:
This is not a good comment. Instead add a docstring describing the problem this is testing, and perhaps refer to this pull request number.